Galera, atualizem esse PR por favor, o build do deploy falhou com os seguintes erros:

4:21:52 PM: src/pages/tests/Flows.test.tsx(6,3): error TS6133: 'fireEvent' is declared but its value is never read.
4:21:52 PM: src/pages/tests/Flows.test.tsx(7,3): error TS6133: 'waitFor' is declared but its value is never read.

removam a importação dessas variáveis que não são utilizadas e façam um push novamente.

--- a/src/pages/__tests__/ViewProcess.test.tsx
+++ b/src/pages/__tests__/ViewProcess.test.tsx
@@ -1,11 +1,5 @@
 import { describe, expect } from "vitest";
-import {
-  act,
-  render,
-  screen,
-  fireEvent,
-  waitFor,
-} from "@testing-library/react";
+import { act, render, screen } from "@testing-library/react";
 import { MemoryRouter } from "react-router-dom";
 import { rest } from "msw";
 import { setupServer } from "msw/node";
@@ -95,26 +89,21 @@ describe("ViewProcess page", async () => {
     expect(screen).toMatchSnapshot();
   });
 
+  it("shows process text content correctly", () => {
+    expect(screen.getByText("(12345678912345678915)")).not.toBe(null);
+    expect(screen.getByText("Processo - Processo NS")).not.toBe(null);
+    expect(screen.getByText("Não iniciado")).not.toBe(null);
+    expect(screen.getByText("Status:")).not.toBe(null);
+    expect(screen.getByText("Fluxo:")).not.toBe(null);
+    expect(screen.getByText("Fluxo 1")).not.toBe(null);
+    expect(screen.getByText("Prioridade Legal:")).not.toBe(null);
+    expect(screen.getByText("Não tem")).not.toBe(null);
+    expect(screen.getByText("Voltar aos Processos")).not.toBe(null);
+    expect(screen.getByText("Iniciar Processo")).not.toBe(null);
+  });
+
   it("shows staged name and due date correctly", () => {
-    expect(screen.findByText("etapaabcdef")).not.toBe(null);
+    expect(screen.findByText("etapaA")).not.toBe(null);
     expect(screen.findByText("Vencimento")).not.toBe(null);
-  });
-
-  it("opens and closes the creation modal correctly", async () => {
-    const ReturnStageButton = await screen.getByText("Retroceder Etapa");
-
-    await act(async () => {
-      await fireEvent.click(ReturnStageButton);
-    });
-
-    const closeModalButton = await screen.getByText("Cancelar");
-
-    await act(async () => {
-      await fireEvent.click(closeModalButton);
-    });
-  });
-
-  await waitFor(() => {
-    expect(screen.queryByText("ReturnModal")).toBeNull();
   });
 });

--- a/src/utils/mocks.ts
+++ b/src/utils/mocks.ts
@@ -39,19 +39,19 @@ export const mockedUnits = [
 export const mockedStages = [
   {
     idStage: 1,
-    name: "etapaabcdef",
+    name: "etapaA",
     duration: 5,
     idUnit: 1,
   },
   {
     idStage: 2,
-    name: "b",
+    name: "etapaB",
     duration: 5,
     idUnit: 1,
   },
   {
     idStage: 3,
-    name: "c",
+    name: "etapaC",
     duration: 5,
     idUnit: 1,
   },
@@ -68,37 +68,26 @@ export const mockedFlow = {
 export const mockedFlows = [
   {
     idFlow: 1,
-    name: "Fluxo 1",
+    name: "FirstFlow",
     idUnit: 1,
     stages: [1, 2],
     sequences: [{ from: 1, commentary: "Comentário", to: 2 }],
   },
   {
     idFlow: 2,
-    name: "Fluxo 2",
+    name: "SecondFlow",
     idUnit: 1,
     stages: [1, 2],
     sequences: [{ from: 1, commentary: "Comentário", to: 2 }],
   },
   {
     idFlow: 3,
-    name: "Fluxo 3",
+    name: "ThirdFlow",
     idUnit: 1,
     stages: [1, 2],
     sequences: [{ from: 1, commentary: "Comentário", to: 2 }],
   },
 ];
-
-export const mockedNotStartedProcess = {
-  effectiveDate: null,
-  idFlow: 1,
-  idPriority: 0,
-  idStage: null,
-  idUnit: 1,
-  nickname: "Apelido do Processo",
-  record: "12345678912345678915",
-  status: "notStarted",
-};
 
 export const mockedProcesses = [
   {
@@ -167,3 +156,32 @@ export const mockedPriorities = [
     description: "Réu Preso",
   },
 ];
+
+export const mockedNotStartedProcess = {
+  effectiveDate: null,
+  idFlow: 1,
+  idPriority: 0,
+  idStage: null,
+  idUnit: 1,
+  nickname: "Processo NS",
+  record: "12345678912345678915",
+  status: "notStarted",
+};
+
+export const mockedInProgressProcess = {
+  effectiveDate: "2023-06-21T01:00:54.109Z",
+  idFlow: 2,
+  idPriority: 0,
+  idStage: 1,
+  idUnit: 1,
+  nickname: "Processo NS",
+  record: "12345678901234567881",
+  status: "inProgress",
+  progress: [
+    {
+      idStage: 1,
+      entrada: "2023-06-21T01:00:54.110Z",
+      vencimento: "2023-06-22T01:00:54.110Z",
+    },
+  ],
+};

